ts/components/screens/ListItemComponent.tsx

```diff
@@ -42,7 +42,8 @@ const styles = StyleSheet.create({
   flexColumn: {
     flexDirection: "column",
     justifyContent: "space-between",
-    flexGrow: 1
```

Comment:
Did you try to fix this issue by only removing **flexGrow: 1** and adding **flex: 1**?
I'm quite sure all other changes are no needed

Reply:
I didn't try it because I thought that maybe that property could be needed in other lists.
I will try now.

Reply:
Yes, it's important to check if these changes could have bad side effects in other lists 👍

Reply:
Ok, it worked. There don't seem to be problems in the other lists like messages list, preferences list and services list.